installer/kickstart.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
also any sh should do, though the installer later is bash only

Reply:
Below, It checks the availablity of bash and its version to make sure it will run